app/src/androidTest/java/com/example/loginsesame/TestApplicationLogin.kt

```diff
@@ -56,7 +83,7 @@ class TestApplicationLogin {
     }
 
     @Test
-    fun userPressesBackButton() {
+    fun userClicksBackButton() {
```

Comment:
Test will not run if soft-keyboard is visible, close keyboard with espresso before pressing back

Reply:
this test doesn't open the keyboard.
It just presses the hardware back button which is always visible. I have added it to the other tests though